### PR TITLE
Mention no images added to repository

### PR DESCRIPTION
**Pull request by LysandreJik**
Mention that images shouldn't be added to the repository as it will otherwise significantly weigh down the repository.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -275,6 +275,10 @@ Follow these steps to start contributing:
    CircleCI does not run the slow tests, but github actions does every night!
 6. All public methods must have informative docstrings that work nicely with sphinx. See `modeling_bert.py` for an
    example.
+7. Due to the rapidly growing repository, it is important to make sure that no file that would significantly weigh down
+   the repository are added. This includes images, videos and other non-text files. We prefer to leverage a hf.co hosted `dataset` like
```
Comment (NielsRogge) [inline CONTRIBUTING.md:279]:
```suggestion
7. Due to the rapidly growing repository, it is important to make sure that no files that would significantly weigh down the repository are added. This includes images, videos and other non-text files. We prefer to leverage a hf.co hosted `dataset` like
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -275,6 +275,10 @@ Follow these steps to start contributing:
...
+   the ones hosted on [`hf-internal-testing`](https://huggingface.co/hf-internal-testing) in which to place these files and reference
```
Comment (NielsRogge) [inline CONTRIBUTING.md:280]: FYI the repo I use for documentation images is called "documentation-images": https://huggingface.co/datasets/huggingface/documentation-images

**Review by sgugger — APPROVED** [review]
Thanks for adding this note. As said on another PR, I think it would be more efficient to couple this with a migration of existing images to the same dataset and completely remove the `imgs` folder, this way no one will be tempted to add an image.

**Comment by LysandreJik** [comment]
WIll merge this as a first step!

**Comment by stas00** [comment]
Merged too soon? We need a concrete destination for images

**Comment by stas00** [comment]
but it's not part of this PR that was merged.

**Pull request merged**